docs/containers-registries.d.5.md

```diff
@@ -84,7 +84,7 @@ described above.  The configuration section is a YAML mapping, with the followin
    This key is optional; if it is missing, no signature storage is defined (no signatures
    are download along with images, adding new signatures is possible only if `sigstore-staging` is defined).
 
-- `use-cosign-attachments` specifies whether Cosign image attachments (signatures, attestations and the like) are going to be read/written along with the image.
+- `use-sigstore-attachments` specifies whether sigstore image attachments (signatures, attestations and the like) are going to be read/written along with the image.
```

Comment:
@rhatdan @vrothberg If this should be `use-attachments` or something else, please speak up.

(The `s/sigstore/lookaside/` changes will be a separate PR.)